common.js

```diff
@@ -4432,6 +4431,524 @@ const allSoloCars = {
+      'all': ['dsp']
+    },
+    'Shelby Charger Turbo': {
+      'all': ['dsp']
+    },
+    'Spirit (4-cyl Turbo & V6)': {
+      'all': ['dsp']
+    },
+    'SRT-4': {
+      'all': ['dsp']
+    },
+    'Sundance Turbo': {
+      'all': ['dsp']
+    },
+    'Barracuda': {
+      '1965': ['cp', 'esp'],
+      '1966': ['cp', 'esp'],
+      '1967': ['cp', 'esp'],
+      '1968': ['cp', 'esp'],
+      '1969': ['cp', 'esp']
+    },
+    'Dart, Duster': {
+      'all': ['esp']
+    },
+    'Valiant (A-body)': {
+      '1963': ['esp'],
+      '1964': ['esp'],
+      '1965': ['esp'],
+      '1966': ['esp'],
+      '1967': ['esp'],
+      '1968': ['esp'],
+      '1969': ['esp'],
+      '1970': ['esp'],
+      '1971': ['esp'],
+      '1972': ['esp'],
+      '1973': ['esp'],
+      '1974': ['esp'],
+      '1975': ['esp'],
+      '1976': ['esp']
+    },
+    'Barracuda & Challenger (E-body)': {
+      '1970': ['esp'],
+      '1971': ['esp'],
+      '1972': ['esp'],
+      '1973': ['esp'],
+      '1974': ['esp']
+    },
+    'Challenger': {
+      '2008': ['esp'],
+      '2009': ['esp'],
+      '2010': ['esp'],
+      '2011': ['esp'],
+      '2012': ['esp'],
+      '2013': ['esp']
+    },
+    'Challenger (6-cyl & V8, NOC)': {
+      'all': ['esp']
+    },
+    'Charger': {
+      '2006': ['esp'],
+      '2007': ['esp'],
+      '2008': ['esp'],
+      '2009': ['esp'],
+      '2010': ['esp'],
+      '2011': ['esp'],
+      '2012': ['esp'],
+      '2013': ['esp']
+    },
+    'Laser (FWD)': {
+      'all': ['esp']
+    },
+    'Stealth (non-turbo)': {
+      'all': ['esp']
+    },
+    'Dakota': {
+      '1997': ['esp'],
+      '1998': ['esp'],
+      '1999': ['esp'],
+      '2000': ['esp'],
+      '2001': ['esp'],
+      '2002': ['esp'],
+      '2003': ['esp'],
+      '2004': ['esp']
+    },
+    'Acclaim (4-cyl non-turbo)': {
+      'all': ['fsp']
+    },
+    'Arrow 1600, 2000, & 2600': {
+      'all': ['fsp']
+    },
+    'Champ (non-turbo, all)': {
+      'all': ['fsp']
+    },
+    'Colt (non-turbo, FWD)': {
+      'all': ['fsp']
+    },
+    'Colt (8v non-turbo)': {
+      'all': ['fsp']
+    },
+    'Colt (1600 & 2000, RWD)': {
+      'all': ['fsp']
+    },
+    'Daytona (non-turbo)': {
+      'all': ['fsp']
+    },
+    'Horizon, TC3, & Turismo (1.7L,1.8L,& 2.2L)': {
+      'all': ['fsp']
+    },
+    'Laser (non-turbo)': {
+      '1989': ['fsp'],
+      '1990': ['fsp'],
+      '1991': ['fsp'],
+      '1992': ['fsp'],
+      '1993': ['fsp'],
+      '1994': ['fsp'],
+      '1995': ['fsp'],
+      '1996': ['fsp'],
+      '1997': ['fsp'],
+      '1998': ['fsp'],
+      '1999': ['fsp']
+    },
+    'Omni, 024, & Charger': {
+      'all': ['fsp']
+    },
+    'Rampage (2.2L)': {
+      'all': ['fsp']
+    },
+    'Sapporo (1600, 2000, & 2600)': {
+      'all': ['fsp']
+    },
+    'Shelby (2.2L non-turbo)': {
+      '1983': ['fsp'],
+      '1984': ['fsp']
+    },
+    'Spirit (4-cyl non-turbo)': {
+      'all': ['fsp']
+    },
+    '300 (all) ': {
```

Comment:
This should be merged with the existing 300 models you have above. Remove this model and simply add `cp` to every year of both the V6 and V8 models you already have.